bag2-loader/src/main/java/nl/b3p/brmo/bag2/loader/cli/BAG2LoadOptions.java

```diff
@@ -18,9 +18,13 @@ public class BAG2LoadOptions {
       negatable = true)
   boolean dropIfExists;
 
+  @CommandLine.Option(names = "--geo-filter", paramLabel = "<wkt>")
```

Comment:
documentatie van deze parameter ontbreekt in de [README](https://github.com/B3Partners/brmo/blob/master/bag2-loader/README.md)